Add in update to AWS Lambda guide to make ExpressReceiver config more clear (#1649)
slackapi/bolt-js: docs/_deployments/aws-lambda.md

const { App, AwsLambdaReceiver } = require('@slack/bolt');
```

> > 💡  If you are planning on implementing authentication with OAuth, as of today you need to use the [`ExpressReceiver`](https://github.com/slackapi/bolt-js/blob/main/src/receivers/ExpressReceiver.ts). Additionally, when using other receivers with that are not the `AwsLambdaReceiver`, please note that the `processBeforeResponse: true` property is required to be passed in to your receiver initialization.

Reviewer: ```suggestion
> 💡  If implementing authentication with OAuth, you must use the [`ExpressReceiver`](https://github.com/slackapi/bolt-js/blob/main/src/receivers/ExpressReceiver.ts). Please note that when using `ExpressReceiver`, the `processBeforeResponse: true` property is required during initialization to avoid latency issues.
```

I had a look at the original conversation to guide the suggestion, but could have gotten it wrong. Feel free to ignore! As far as I could tell, the suggestion we make (and that the developer used) was `ExpressReceiver`, so I kept the language around that specific case.

Reviewer 2: I think this makes it even more clear. Thank you @misscoded

Author: This is perfect! Thanks so much @misscoded 🙌
